flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesClusterDescriptorTest.java

```diff
@@ -147,4 +173,30 @@ public void testKillCluster() throws Exception {
 
 		return clusterClient;
 	}
+
+	private void mockRestServiceWithLoadBalancer(@Nullable String hostname, @Nullable String ip) {
+		final String restServiceName = KubernetesUtils.getRestServiceName(CLUSTER_ID);
+
+		final String path = String.format("/api/v1/namespaces/%s/services/%s", NAMESPACE, restServiceName);
+		server.expect()
+			.withPath(path)
+			.andReturn(200, buildMockRestService(hostname, ip))
+			.always();
+	}
+
+	private Service buildMockRestService(@Nullable String hostname, @Nullable String ip) {
```

Comment:
`@Nullable` could be removed.

Reply:
Fixed by [d78c1b1](https://github.com/apache/flink/pull/11233/commits/d78c1b1b282a3d7fcfc797bf909dca2b5f2264e4)